docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -398,25 +426,27 @@ quarkus.oidc.user-info-path=/protocol/openid-connect/userinfo
 quarkus.oidc.introspection-path=/protocol/openid-connect/tokens/introspect
 ----
 
-=== Token Propagation
+[[oidc-token-propagation]]
+=== Token propagation
 
-Please see xref:security-openid-connect-client-reference.adoc#token-propagation[Token Propagation] section about the Bearer access token propagation to the downstream services.
+For information about bearer access token propagation to the downstream services, see the xref:security-openid-connect-client-reference.adoc#token-propagation[Token propagation] section of the Quarkus "OpenID Connect (OIDC) and OAuth2 client and filters reference" guide.
 
 [[oidc-provider-authentication]]
```

Comment:
Likewise, it is not used, and not clear if the existing links will remain working. `oidc-provider-client-authentication` would be correct in any case as it is about the the quarkus OIDC client code authenticating to the OIDC provider to complete its actions

Reply:
Removed anchor again, per your guidance.